src/main/resources/data/miningmaster/advancements/miningmaster/the_collector.json

```diff
@@ -91,6 +91,42 @@
         ]
       }
     },
+    "miningmaster:divine_beryl": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "miningmaster:divine_beryl"
+            ]
+          }
+        ]
+      }
+    },
+    "miningmaster:spider_kunzite": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "miningmaster:spider_kunzite"
+            ]
+          }
+        ]
+      }
+    },
+    "miningmaster:unbreaking_iolite": {
+      "trigger": "minecraft:inventory_changed",
+      "conditions": {
+        "items": [
+          {
+            "items": [
+              "miningmaster:unbreaking_iolite"
+            ]
+          }
+        ]
+      }
+    },
```

Comment:
You added these, but didn't actually add them to the requirements for the advancement.